env.example

```diff
@@ -113,6 +113,9 @@ ETHERPAD_SKIN_VARIANTS="super-light-toolbar super-light-editor light-background
 # Enable authentication
 #ENABLE_AUTH=1
 
+# Enable authentication in Jicofo: defaults to ENABLE_AUTH
+#JICOFO_ENABLE_AUTH=1
```

Comment:
No need to document it here. Do so in the handbook please. The sample env is just for the essentials.